install.ps1

```diff
@@ -318,8 +340,8 @@ if ($nochecks -eq $false) {
 
 
   $osversion = (Get-WmiObject -class Win32_OperatingSystem).BuildNumber
-  if (-Not (($osversion -eq 18363) -or ($osversion -eq 18361) -or ($osversion -eq 17763) -or ($osversion -eq 17134) -or ($osversion -eq 19041) )){
-    Write-Host "`t[ERR] Windows version $osversion is not has not been tested, please use Windows 10 version 1803, 1809, 1903, 1909, or 2004." -ForegroundColor Yellow
+  if (-Not (($osversion -eq 18363) -or ($osversion -eq 18361) -or ($osversion -eq 17763) -or ($osversion -eq 17134) -or ($osversion -eq 19041) -or ($osversion -eq 19042) -or ($osversion -eq 19043) )){
```

Comment:
One way to make this easier to read / maintain:

```suggestion
  $valid_versions = @(18363, 18361, 17763, 17134, 19041, 19042, 19043)
  if ($osversion -notin $valid_versions) {
```

Examples of how it works:
![Screen Shot 2021-07-13 at 10 31 11](https://user-images.githubusercontent.com/20993114/125470466-ecee3843-8ae8-4a2b-8572-f673fb227b5a.png)